newIDE/app/src/Utils/GDevelopServices/Build.js

```diff
@@ -61,6 +61,29 @@ export const getBuildArtifactUrl = (
   return `https://builds.gdevelop-app.com/${build[keyName]}`;
 };
 
+export const getWebBuildGameFolderUrl = (buildId: string): string => {
+  return `https:/games.gdevelop-app.com/game-${buildId}`;
+};
+
+export const getWebBuildThumbnailUrl = (
+  project: gdProject,
+  buildId: string
+): string => {
+  // TODO use some helper methods to do this?
+  const path = project
+    .getPlatformSpecificAssets()
+    .get('liluo', `thumbnail`)
+    .split('\\');
+  if (path.length < 1) {
+    return '';
+  }
+  const fileName = path[path.length - 1];
+  if (!fileName) {
+    return '';
+  }
+  return `https:/games.gdevelop-app.com/game-${buildId}/${fileName}`;
```

Comment:
The majority of this function is just getting the file name out of a path. This can simply be done using the `path` module, which is easily (and IIRC already is by default by CRA) polyfilled to work just fine on the browser. This would make the function cleaner to read: to be clean, a function must do what you expect it to do from its name. This function is more of a `basename` than a `getThumbnailURL`, making it take more time to understand than it needs to. 
```suggestion
  import { basename } from "path";
  return `https:/games.gdevelop-app.com/game-${buildId}/${basename(path)}`;
```

Reply:
Thanks, that's partially answer my TODO comment.